radicle-cli/src/commands/patch/checkout.rs

```diff
@@ -1,38 +1,85 @@
-use crate::terminal as term;
 use anyhow::anyhow;
-use radicle::cob::patch::{self, PatchId};
-use radicle::git::{self, RefString};
+
+use radicle::cob::patch;
+use radicle::cob::patch::{Patch, PatchId};
+use radicle::git;
+use radicle::git::RefString;
 use radicle::storage::git::Repository;
+use radicle::storage::ReadRepository;
+
+use crate::terminal as term;
 
 pub fn run(
-    storage: &Repository,
-    git_workdir: &git::raw::Repository,
+    stored: &Repository,
```

Comment:
Why the rename, out of interest?

Reply:
`storage` is confusing when it doesn't refer to `Storage`, no?

I think a nice convention if there are two repositories is `stored` for the stored copy and `working` for the working copy. That was the idea.